plugins/syslog.yaml

```diff
@@ -37,7 +37,7 @@ parameters:
 # Set Defaults
 # {{$listen_address := default "0.0.0.0:514" .listen_address}}
 # {{$connection_type := default "udp" .connection_type}}
-# {{$protocol := default "rfc5424" .protocol}}
+# {{$protocol := default "rfc5424 (IETF)" .protocol}}
```

Comment:
This looks to set the default to a value that will cause the syslog parser to fail. We had fixed this previously.